dbconn/factory.go

```diff
@@ -218,7 +229,19 @@ func (f factory) Password(cfg blip.ConfigMonitor) (PasswordFunc, error) {
 			return nil, err
 		}
 		token := aws.NewAuthToken(cfg.Username, cfg.Hostname, awscfg)
-		return token.Password, nil
+		return func(ctx context.Context) (*Credentials, error) {
+			passwd, err := token.Password(ctx)
+
```

Comment:
Remove this empty line. The `if err != nil {` boilerplate is pretty much always next to the call that made `err`.